src/components/ContactCard/ContactForm.jsx

```diff
@@ -66,6 +70,11 @@ const fields = [
   }
 ];
 
+const initialFieldValues = fields.reduce((initialValues, { name, isArray }) => {
+  initialValues[name] = isArray ? [undefined] : undefined;
+  return initialValues;
+}, {});
+
```

Comment:
We are now using final-form's `FieldArray`, and arrays of field start empty by default. So we must init those fields as an array containing one element.

Reply:
[Style] It seems weird to do it that way, but I don't have something better to propose. However, if usefull: Shouldn't you put your comment directly in the code instead of a github comment ?

Reply:
True, the line is confusing enough that it warrants an explanation in the code.

Reply:
Ouais, c'est vraiment weird!